src/components/shared/translations.js

```diff
@@ -9,7 +9,13 @@ const de = {
   save: "Speichern",
   cancel: "Abbrechen",
   choose: "Auswählen",
-  tagCreate: "Tag erstellen"
+  tagCreate: "Tag erstellen",
+  entrySelected: "Eintrag ausgewählt",
+  entriesSelected: "Einträge ausgewählt",
+  ariaExpandAll: "Alle Elemente ausklappen",
+  ariaSelectAll: "Alle Elemente auswählen",
+  itemLockedForSelection: "Element kann nicht ausgewählt werden",
+  explanationNoentries: "Es sind keine Einträge vorhanden."
```

Comment:
we should order this document alphabetically too.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/429/commits/d4c7d53e26757f641a9ec7bfff88929431dc8a9b